### DISCUSSION TITLE
Gating Strategy and Session class related queries

### DISCUSSION THREAD

**Discussion by jainaditi0204** [whitews/FlowKit#119] [General]
Hi Scott,

How are you?
I was wondering if there's any way to visualize gated events for a given sample? I am applying a gating strategy from my gatingml file to a few index-sorted sample files (as well as the normal FCS files), and I can easily retrieve the report with percentages and everything, but some values are inconsistent with the observations, and so I wanted to visualize the gated events? 
Also, is there any way the user can apply a gating strategy to a session (or import a session into the said strategy)? In my case, I can't extract any gating information from the samples within the session, so I am trying to apply the gates from the gatingML to those samples (as a session). I did it for one sample at a time, and it works perfectly. 
I look forward to hearing your thoughts! :)

Best, 
Aditi

**Comment by whitews** [comment]
Hi Aditi,

Yes, if you have a gating strategy, you can use it when creating a new sample group within a session, like so:

```
session = fk.Session()
session.add_sample_group('my group name', gating_strategy=strategy_instance)
```

For plotting gated events, you can use the Session method `plot_scatter` and use the optional `gate_name` argument. That method also allows you to set any channels as the x & y axis so you can view gated events pretty much anyway you want.

**Reply by jainaditi0204** [reply]
Hi Scott,

Thank you for your reply. 
I overlooked that method, and I just tried your suggestion, and it works as expected.
Thank you, again. 

Best,
Aditi
